packages/addresses/README.md

```diff
@@ -51,6 +70,39 @@ function handleSubmit() {
 }
 ```
 
+### `getAddressDecoder()`
+
+Returns a decoder that you can use to convert an array of 32 bytes representing an address to the base58-encoded representation of that address. Returns a tuple of the `Base58EncodedAddress` and the offset within the byte array at which the decoder stopped reading.
+
+```ts
+import { getAddressDecoder } from '@solana/addresses';
+
+const addressBytes = new Uint8Array([
+    150, 183, 190, 48, 171, 8, 39, 156, 122, 213, 172, 108, 193, 95, 26, 158, 149, 243, 115, 254, 20, 200, 36, 30, 248,
+    179, 178, 232, 220, 89, 53, 127,
+]);
+const addressDecoder = getAddressDecoder();
+const [address, offset] = addressDecoder.decode(address); // [B9Lf9z5BfNPT4d5KMeaBFx8x1G4CULZYR1jA2kmxRDka, 32]
+```
+
+### `getAddressEncoder()`
+
+Returns a codec that you can use to encode a base58-encoded address to a byte array.
```

Comment:
```suggestion
Returns an encoder that you can use to encode a base58-encoded address to a byte array.
```

Nit but we do have a separate `getAddressCodec` so I think calling the encoder a codec is a bit confusing